src/libaktualizr/config/config_test.cc

```diff
@@ -169,19 +169,35 @@ TEST(config, TomlConsistentNonempty) {
   EXPECT_EQ(conf_str1, conf_str2);
 }
 
+static std::vector<std::string> generate_multi_config(TemporaryDirectory &temp_dir) {
+  Utils::writeFile((temp_dir / "a_dir/a.toml"), std::string("[storage]\npath = \"path_a\"\n\n[pacman]\nos = \"os_a\""));
```

Comment:
For a bit more readability, it could use the same style as in the `TwoTomlCorrectness` test, where new lines are reflected in the source.

Reply:
OK, thanks, I'll modify it.